src/tests/baseservices/typeequivalence/functionpointer/Functionpointer.cs

```diff
@@ -0,0 +1,46 @@
+using System;
```

Comment:
@fanyang-mono nit: "typeequivalence" not general "type equivalence". it's a special thing that CoreCLR implements for COM interop https://learn.microsoft.com/en-us/dotnet/framework/interop/type-equivalence-and-embedded-interop-types.

A better place for this test would be in src/tests/Loader/classloader/Casting maybe

Reply:
Ah! Will move it.